src/material/schematics/ng-generate/mdc-migration/rules/components/chips/chips-template.ts

```diff
@@ -0,0 +1,124 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import * as compiler from '@angular/compiler';
+import {TemplateMigrator, Update} from '../../template-migrator';
+import {replaceStartTag, replaceEndTag, visitElements} from '../../tree-traversal';
+
+export class ChipsTemplateMigrator extends TemplateMigrator {
+  chipInputAttrs: compiler.TmplAstBoundAttribute[] = [];
+
+  getUpdates(node: compiler.TmplAstElement): Update[] {
+    if (node.name === 'mat-chip-list') {
+      return this._buildChipListUpdates(node);
+    }
+
+    if (node.name === 'input') {
+      this._storeChipRefs(node);
+    }
+
+    return [];
+  }
+
+  override onComplete(): void {
```

Comment:
Optional suggestion to follow the idea of naming things what they are [supposed to do](https://github.com/angular/components/blob/master/CODING_STANDARDS.md#methods) 

```suggestion
  override reset(): void {
```